platform/platform-app/src/main/resources/features.xml

```diff
@@ -94,16 +95,25 @@
         <feature version="${camel.version}">camel-core</feature>
         <bundle>mvn:org.apache.camel/camel-spring/${camel.version}</bundle>
     </feature>
+
+    <!--
+        The following two bulletted changes were made to allow camel-amqp to start without resolution conflicts
```

Comment:
Me too, but Intellij didn't like it so I switched it to `bulletted` but Intellij didn't like that either. But okay, that's one vote for `bulleted` and none for `bulletted` so I'll switch it back! 👍